pandas/plotting/_core.py

```diff
@@ -914,6 +914,12 @@ def __call__(self, *args, **kwargs):
         --------
         matplotlib.pyplot.plot : Plot y versus x as lines and/or markers.
 
+        Notes
+        -----
+        Where pandas fails to infer the frequency information the automatic tick
+        resolution adjustment can be suppressed by the parameter:
+        `x_compact=True`  (e.g., `s.plot(x_compact=True)`).
```

Comment:
These should all be in double backticks and should be `x-compat`, not `x-compact`.